molecule/default/prepare.yml

```
@@ -18,13 +18,14 @@
 - name: Wait for systemd to complete initialization (Fedora)
   hosts: os_Fedora
   tasks:
-    - name: Wait for systemd to complete initialization # noqa 303
+    - name: Wait for systemd to complete initialization # noqa command-instead-of-module
       ansible.builtin.command: systemctl is-system-running
       register: systemctl_status
       until: "'running' in systemctl_status.stdout"
       retries: 30
       delay: 5
       when: ansible_service_mgr == 'systemd'
+      changed_when: "'running' in systemctl_status.stdout"
```

Clean up/remove unnecessary quotes:
```suggestion
      until: 'running' in systemctl_status.stdout
      retries: 30
      delay: 5
      when: ansible_service_mgr == "systemd"
      changed_when: "running" in systemctl_status.stdout
```

---

`until` definitely needs the quotes, I will test the changed_when but I believe it was failing when I didn't have that in quotes at first

---

Well, not sure if it would run or not but yamllint chokes on it when those lines aren't enclosed in double quotes

---

OK